src/server/handlers/openapi/mojaloop/consentRequests/{ID}.ts

```diff
@@ -20,14 +20,29 @@
 
  * Google
  - Steven Wijaya <stevenwjy@google.com>
+ - Abhimanyu Kapur <abhi.kapur09@gmail.com>
  --------------
  ******/
 
 import { Request, ResponseToolkit } from '@hapi/hapi'
 import { Handler, Context } from 'openapi-backend'
 import { logger } from '~/shared/logger'
+import { consentRepository } from '~/repositories/consent'
+import { ConsentStatus } from '~/models/consent'
 
-export const put: Handler = async (context: Context, request: Request, h: ResponseToolkit) => {
+export const put: Handler = async (
+  context: Context,
+  request: Request,
+  h: ResponseToolkit
+) => {
   logger.logRequest(context, request, h)
+
+  const id = context.request.params.id as string
```

Comment:
shouldn't it be `params.ID` instead of `params.id` (based on the open api definition)? Or perhaps have you checked whether this one works? iirc the variables are case sensitive ...

Reply:
ah i was under the assumption they weren't sensitive, thanks for the catch